lib/fluent/plugin/filter_sanitizer.rb

```diff
@@ -339,6 +360,22 @@ def sanitize_keywords_val(v, keywords, prefix)
         return line.join(" ")
       end
 
+      def sanitize(str)
```

Comment:
I think this method is slow. We can create callable object in `configure`

```
# Inside configure
@sanitize_func =
  case @hash_scheme
  when :md5
    Proc.new { |str| Digest::MD5.hexdigest(@salt + str) }
  when :sha1
    Proc.new { |str| Digest::SHA1.hexdigest(@salt + str) }
  # Put same code for other schemas
  end

# Inside sanitize_ipv4
return "IPv4_"+ @sanitize_func.call(str)
```

Reply:
@repeatedly  I have made the relevant changes